plugin/pkg/admission/admit/admission.go

```diff
@@ -37,12 +38,12 @@ var _ admission.MutationInterface = AlwaysAdmit{}
 var _ admission.ValidationInterface = AlwaysAdmit{}
 
 // Admit makes an admission decision based on the request attributes
-func (AlwaysAdmit) Admit(a admission.Attributes) (err error) {
+func (AlwaysAdmit) Admit(_ request.Context, a admission.Attributes) (err error) {
```

Comment:
We can also add `audit` to admission.Attributes.
But `audit` is not an attribute.

I hope I am in the right direction.